### DISCUSSION TITLE
Can you add "last modified date" to search result card layout?

### DISCUSSION THREAD

**Discussion by janetslei** [microsoft-search/pnp-modern-search#2691] [Q&A] [answered]
Can you add "last modified date" to search result card layout? 
I'm using a query that filters on all pages in the site. I was wondering how can you add "last modified date" or "published date" to the search results web part UI? 
I was able to add the "last modified date" to the search filters by just mapping it to a managed property, but is there a way to show date in cards? The author field shows both Author name and Created date.

thank you for your help!

**Comment by kasperbolarsen** [comment, marked as answer]
Hi @janetslei 
Sure, like this 
![image](https://user-images.githubusercontent.com/20593570/212193652-9c5f22cd-e081-46dd-bff9-150e98929479.png)


All it takes it that you add LastModifiedTime  in "selected properties", then update "Edit layout slots for this data source" with LastModifiedTime:
![image](https://user-images.githubusercontent.com/20593570/212193980-d684b2d9-fbff-4ee7-988d-e1d789a8a02b.png)

and finally you will have to add or replace the Date placeholder in the "Managed card fields" with this 
Last updated: {{getDate (slot item @root.slots.LastModifiedTime) 'LL'}}

**Reply by janetslei** [reply]
can you have the Date field without the Author on the card or are they a package deal?

**Reply by kasperbolarsen** [reply]
In the "Cards" layout the Date field is integrated with the Author field, so the best you can do will be something like 
![image](https://user-images.githubusercontent.com/20593570/212260850-d04d55f4-10aa-4a19-a5e0-47cbb47a628c.png)


However you can customize the layout templates using either Handlebars/HTML or Adaptive Cards as per https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/

**Reply by janetslei** [reply]
I figured, thank you so much for your help!
